src/components/EtaxForm/configs/AddressFields.config.js

```diff
@@ -7,46 +7,95 @@ import _ from "lodash"
  */
 const addressFields = [
   {
-    id: "houseNo",
-    label: "House No.",
-    helperText: "Your House Number",
+    id: "postCode",
+    label: "รหัสไปรษณีย์",
+    helperText: "ex. 11120, 10520",
+    margin: "normal",
+    required: false,
+  },
+  {
+    id: "buildingName",
+    label: "ชื่ออาคาร",
+    helperText: "ex. อาคารเรียนรวมเทคโนโลยีสารสนเทศ",
+    margin: "normal",
+    required: false,
```

Comment:
Tip: If most of these fields have similar properties, such as `required: false` and `margin: "normal"`, you can use constructs such as `map` to make your life a bit easier! 💃 

For instance:

```
fields.map(field => ({
  required: false,
  margin: "normal",
  ...field
})
```

By spreading the field item after the common properties, so that your array can override those common properties, such as `required: true` 💯

Reply:
Thanks for your suggestion.